src/css/custom.css

```diff
@@ -0,0 +1,8 @@
+/**
+ * WIKILINK CLASSES
+ */
+/*.wikilink {}*/
+.wikilink-new,
```

Comment:
This can be simplified to `a.wikilink-new`, which will stop the `a:hover` color from overriding the `.wikilink-new` color while hovering.